app/views/devise/mailer/reset_password_instructions.html.slim

```diff
@@ -0,0 +1,7 @@
+.row
+  .col-xs-12.col-md-8.col-md-offset-2
+    p Hello = @resource.email!
```

Comment:
@DRot88 Looks good to me. Once this ^ is changed. Good to merge. Awesome work. :star: